packages/api-explorer/src/components/Response/index.jsx

```diff
@@ -11,8 +11,24 @@ const ResponseBody = require('./ResponseBody');
 const showCodeResults = require('../../lib/show-code-results');
 
 const Oas = require('../../lib/Oas');
+const CopyCode = require('../CopyCode');
 
 const { Operation } = Oas;
+const ctaContainerStyle = {
```

Comment:
why cta?

Reply:
I used it to indicate the whole "call to action" button inside the container.
Maybe ContainerCtaStyle is better.